package.json

```diff
@@ -17,35 +17,34 @@
   "author": "",
   "license": "ISC",
   "devDependencies": {
-    "@types/express": "^4.17.17",
-    "@types/minimist": "^1.2.2",
+    "@types/express": "^5.0.0",
+    "@types/minimist": "^1.2.5",
     "@types/object-hash": "^3.0.6",
-    "@types/papaparse": "^5.3.7",
-    "@types/uuid": "^9.0.1",
-    "@typescript-eslint/eslint-plugin": "^5.58.0",
-    "@typescript-eslint/parser": "^5.58.0",
-    "eslint": "^8.38.0",
-    "eslint-config-prettier": "^8.8.0",
-    "eslint-config-standard": "^17.0.0",
-    "eslint-plugin-import": "^2.27.5",
-    "eslint-plugin-n": "^15.7.0",
-    "eslint-plugin-prettier": "^4.2.1",
-    "eslint-plugin-promise": "^6.1.1",
-    "nock": "^14.0.0-beta.7",
-    "nodemon": "^2.0.22",
-    "prettier": "2.8.7",
-    "ts-node": "^10.9.1",
-    "typescript": "^5.0.4",
-    "vitest": "^0.31.1"
+    "@types/papaparse": "^5.3.15",
+    "@types/uuid": "^10.0.0",
+    "@typescript-eslint/eslint-plugin": "^8.19.1",
+    "@typescript-eslint/parser": "^8.19.1",
+    "eslint": "^9.17.0",
+    "eslint-config-prettier": "^9.1.0",
+    "eslint-plugin-import": "^2.31.0",
+    "eslint-plugin-n": "^17.15.1",
+    "eslint-plugin-prettier": "^ 5.2.1",
+    "eslint-plugin-promise": "^7.2.1",
+    "nock": "^13.5.6",
+    "nodemon": "^3.1.9",
+    "prettier": "3.4.2",
+    "ts-node": "^10.9.2",
+    "typescript": "^5.7.2",
+    "vitest": "^2.1.8"
   },
   "dependencies": {
-    "dotenv": "^16.0.3",
-    "express": "^4.18.2",
+    "dotenv": "^16.4.7",
+    "express": "^4.21.2",
     "minimist": "^1.2.8",
     "object-hash": "^3.0.0",
     "papaparse": "^5.4.1",
-    "uuid": "^9.0.0",
-    "winston": "^3.11.0",
-    "winston-daily-rotate-file": "^4.7.1"
+    "uuid": "^11.0.4",
+    "winston": "^3.17.0",
+    "winston-daily-rotate-file": "^5.0.0"
```

Comment:
> Starting with version 5.0.0 this module also emits an "error" event for all low level filesystem error cases. Make sure to listen for this event to prevent crashes in your application.